clientupdate/clientupdate_windows.go

```diff
@@ -26,3 +33,25 @@ const certSubjectTailscale = "Tailscale Inc."
 func verifyTailscale(path string) error {
 	return authenticode.Verify(path, certSubjectTailscale)
 }
+
+func launchTailscaleAsGUIUser(exePath string) error {
+	exePath = filepath.Join(filepath.Dir(exePath), "tailscale-ipn.exe")
+
+	var token windows.Token
+	if u, err := user.Current(); err == nil && u.Name == "SYSTEM" {
```

Comment:
Have you tried running this? Is the GUI running again and is it running as the correct user and visible in their logon session?

Reply:
Yes, this works as expected on my windows 11 VM.
While logged in as my Admin-level user the GUI starts up after update in the current logon session, whether running `tailscale.exe update` from Administrator powershell or from `tailscaled.exe` running as SYSTEM.

Reply:
OK, thanks.